docs/config/index.md

```diff
@@ -966,6 +966,62 @@ Open Vitest UI (WIP)
 
 Listen to port and serve API. When set to true, the default port is 51204
 
+### browser
+
+- **Type:** `boolean | 'firefox' | 'chrome' | 'edge' | 'safari'`
+- **Default:** `false`
+- **CLI:** `--browser`, `--browser=<name>`
+
+Run Vitest in a browser. If browser name is specified, Vitest will run tests using `webdriverio`, otherwise it will open your default browser.
```

Comment:
```suggestion
Run Vitest in a browser. If the browser name is specified, Vitest will run tests using `webdriverio`, otherwise, it will open your default browser.
```